express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+let nextId = 1;
+const gradesArray = [];
+let grades = [];
```

Comment:
What is this? It doesn't appear to actually be used. On line 16 you reassign it to something that is not an array! Down there, it should just be declare as a new `const` instead of using this global variable.